packages/google_maps_flutter/test/circle_updates_test.dart

```diff
@@ -73,10 +73,10 @@ void main() {
 
     final Circle addedCircle = platformGoogleMap.circlesToAdd.first;
     expect(addedCircle, equals(c2));
+
     expect(platformGoogleMap.circleIdsToRemove.isEmpty, true);
 
```

Comment:
I think having a test with multiple circles, where we only change a couple of them and compute the updates would be interesting. Same for others too.

Reply:
Cool, I'll will add some tests.